src/lib/log.js

```diff
@@ -0,0 +1,23 @@
+const chalk = require('chalk')
```

Comment:
I'm planning on moving more log messages to a centralized location.
This will add to consistency and we can even lint those as some point.